test/box-upgradability.test.ts

```diff
@@ -47,11 +45,8 @@ describe('Box Upgrade contract', () => {
 
     beforeEach(async () => {
         box = await deployContractWithProxy<Box>('Box')
-        upgradedListener = new EventListener<ActualUpgradedEvent>(
-            box,
-            'Upgraded',
-            (event) => upgradedEvent(event)
-        )
+        // upgradedListener = eventOf(box, 'Upgraded').newListener()
```

Comment:
issue: commented out line,

_Commented out lines don't get into main_ 😉

Reply:
Pardon, it has stuck in stash
fixed